src/scss/layout/_cta.scss

```diff
@@ -8,3 +10,40 @@
   grid-template-columns: minmax(min-content, var(--page));
   justify-content: center;
 }
+
+.hire-us-cta {
+  --reverse-y: 'content' 'image';
+  --media-gap: var(--double-gutter);
+
+  background-image: url('/assets/media/transition-bg.png');
+  background-position: center;
+  background-size: cover;
+  color: var(--text);
+  font-size: revert;
+  padding: var(--double-gutter) var(--page-margin)
+    var(--cta-padding-block-end, 0);
+  margin-block: var(--spacer);
```

Comment:
```suggestion
  margin-block: var(--spacer);
  padding: var(--double-gutter) var(--page-margin)
    var(--cta-padding-block-end, 0);
```

Reply:
fixed in https://github.com/oddbird/oddleventy/pull/808/commits/15b0a3f91ab66ebdb1aaf83febbce025309cdbfe